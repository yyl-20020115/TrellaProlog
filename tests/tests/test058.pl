:-initialization(main).

main :- sub_atom(abc,B,L,A,S),writeq([B,L,A,S]), nl, fail.
main :- nl, sub_atom(abc,B,2,A,S),writeq([B,2,A,S]), nl, fail.
main :- nl, sub_atom(abc,0,2,A,S),writeq([0,2,A,S]), nl, fail.
main :- nl, sub_atom(abc,0,L,A,S),writeq([0,L,A,S]), nl, fail.
main :- nl, sub_atom(abc,B,L,0,S),writeq([B,L,0,S]), nl, fail.
main :- nl, sub_atom(abc,B,2,0,S),writeq([B,2,0,S]), nl, fail.
main.
