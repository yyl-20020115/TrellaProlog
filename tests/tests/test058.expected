[0,0,3,'']
[0,1,2,a]
[0,2,1,ab]
[0,3,0,abc]
[1,0,2,'']
[1,1,1,b]
[1,2,0,bc]
[2,0,1,'']
[2,1,0,c]
[3,0,0,'']

[0,2,1,ab]
[1,2,0,bc]

[0,2,1,ab]

[0,0,3,'']
[0,1,2,a]
[0,2,1,ab]
[0,3,0,abc]

[0,3,0,abc]
[1,2,0,bc]
[2,1,0,c]
[3,0,0,'']

[1,2,0,bc]
